src/dali/plugin/pair_converter/csv/kraken.py

```diff
@@ -254,7 +254,7 @@ def find_historical_bar(self, base_asset: str, quote_asset: str, timestamp: date
 
         base_file: str = f"{base_asset}_OHLCVT.zip"
 
-        self.__logger.info(F"Attempting to load %s from Kraken Google Drive for timestamp={timestamp}.", base_file)
+        self.__logger.info(f"Attempting to load %s from Kraken Google Drive for timestamp={timestamp}.", base_file)
```

Comment:
OK. I think I have to do it the second way because the logger doesn't seem to take f-strings, it has to be the old-style. For some reason, it allows f-string if the old-style is used too.